README.md

```diff
@@ -12,15 +12,39 @@ all the heartbleed.
 
 Conduit is **experimental**. Currently, it _only supports HTTP/2_ and is especially
 well-suited for [gRPC][grpc]. Follow our progress towards production-readiness here and on
-[twitter][twitter].
+[Twitter][twitter].
 
 <!-- TODO add roadmap link -->
 
+## Documentation
+
+View [Conduit docs][conduit-docs] for more a more comprehensive guide to
+getting started.
+
+
+## Getting started with Conduit
+
+1. Install the Conduit CLI `curl https://run.conduit.io/install | sh `
+
+2. Copy `/Users/username/.conduit/bin/conduit` into your PATH
+
+3. Run `conduit install | kubectl apply -f -` to install Conduit
+
+4. Check to make sure you're running the latest version _(0.1.0)_ of Conduit
```

Comment:
I'm a bit wary of hardcoding a version number here, since then we'll need to remember to update it as part of each release. What if we just linked "latest version" to https://github.com/runconduit/conduit/releases?

```
4. Check to make sure you're running the [latest version][releases] of Conduit
```

And then in the refs section, uncomment `<!-- [releases]: https://github.com/runconduit/conduit -->`, and change it to:

```
[releases]: https://github.com/runconduit/conduit/releases
```